ci-operator/jobs/redhat-openshift-ecosystem/community-operators-pipeline-preprod/redhat-openshift-ecosystem-community-operators-pipeline-preprod-main-presubmits.yaml

```diff
@@ -0,0 +1,392 @@
+presubmits:
+  redhat-openshift-ecosystem/community-operators-pipeline-preprod:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
```

Comment:
Perhaps this is the part we need to change to trigger prow on the preprod branches too? Not sure if the branch in zz_generated_metadata matters...

Reply:
Yes this part is auto generated from config/ files, this will be changed automatically